### ISSUE TITLE
Add link to preoject page

### ISSUE THREAD

**Issue by palob** [bil-elmoussaoui/Icon-Requests#26]
such that theme creators know what the reported app is about.

**Comment by bilelmoussaoui** [comment]
Don't know if it's possible to get the link of the project page. Maybe the description in the desktop file could help?

**Comment by palob** [comment]
Yes, that's definitely helpful. Maybe a popup asking the user for input?
